src/UI/UIView.cpp

```diff
@@ -521,6 +521,49 @@ void UIView::draw_ui_callback()
     }
     ImGui::EndTabItem();
   }
+
+  if (ImGui::BeginTabItem("Camera"))
+  {
+
+    ImGui::Text("Points of Interest");
+    
+    if (ImGui::Button("Add")) {
+      m_controller_app.get_trackball_camera().add_camera_poi();
+      m_camera_poi_table_selection = 0;
+    }
+    ImGui::SameLine();
+    if (ImGui::Button("Remove")) {
+      if (m_controller_app.get_trackball_camera().get_camera_poi().size() > 0)
```

Comment:
I don't think you need the `>0` here, it will return `false` if `.size() == 0` anyways